src/app/components/create-hero/create-hero.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit } from '@angular/core';
+import { CreateHeroService } from '../../services/create-hero/create-hero.service';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-create-hero',
+  templateUrl: './create-hero.component.html',
+  styleUrls: ['./create-hero.component.scss']
+})
+export class CreateHeroComponent implements OnInit {
+  public heroNameInput: string;
+  userId = 1;
+
+  constructor(private create: CreateHeroService, public router: Router) { }
+
+  createHero() {
+    if (this.heroNameInput.length <= 10 && this.heroNameInput.length >= 3) {
+      this.create.createNewHero(this.heroNameInput, this.userId.toString()).subscribe(() => {
+        this.router.navigate(['heroes-hall']);
+      },
+        (err) => {
+          console.log(err.Message);
+        });
+    } else {
+      console.log('Username should be 3 to 10 characters');
```

Comment:
there should be "Hero name" instead of Username

Reply:
that's just temporary